### PR TITLE
If status is in LA Draft state check for read only fields. 

### PR DESCRIPTION
**Pull request by elcorbs**

### CODE REVIEW

File: src/UseCase/GenerateReadOnlySchema/index.js [hunk 1]
```diff
@@ -1,23 +1,22 @@
 export default class GenerateReadOnlySchema {
-  execute(data) {
-    return this.generateUISchema(data.properties);
+  execute(data, flag = 'readonly') {
```
Comment (ccp92) [inline src/UseCase/GenerateReadOnlySchema/index.js]: Is this 'readonly' meant to be 'laReadyOnly'?

File: src/Components/ProjectPage/index.js [hunk 2]
```diff
@@ -8,13 +8,19 @@ export default class ProjectPage extends React.Component {
   }
 
   presentProject = async projectData => {
+    let uiSchema = {}
+    if(projectData.status!=="Submitted") {
```
Comment (danielburnley) [inline src/Components/ProjectPage/index.js]: Should this be submitted or should it be `=== "la-draft"` (or whatever the status was for la draft)

Comment (danielburnley) [reply src/Components/ProjectPage/index.js]: Cos we only want to generate a readonly schema if it's an LA draft, Homes England drafts can be fully edited

File: src/UseCase/GenerateReadOnlySchema/index.js (hunk 1 above)
Comment (elcorbs) [reply src/UseCase/GenerateReadOnlySchema/index.js]: readonly is a flag used elsewhere, so i've set it as the default here so it doesn't break other places it is used.

Comment (ccp92) [reply src/UseCase/GenerateReadOnlySchema/index.js]: Cool, sounds good to me!

File: src/Components/ProjectPage/index.js (hunk 2 above)
Comment (elcorbs) [reply src/Components/ProjectPage/index.js]: Yeah you're right, got a bit confused here, changed it.

**Pull request merged**